docs/content/mobile/ios/local-deploy.md

```diff
@@ -42,6 +42,8 @@ You need the Unity Remote app for this. See the [Unity documentation](https://do
     > You don’t need to enter anything in the text field.
 
   1. Play the game on the Simulator.
```

Comment:
These numbers are rendering oddly in GitHub - are they OK in improbadoc locally?

Reply:
Yeah, it looked fine when I've tested it locally. I'll double-check